### PR TITLE
Configure PSA for Eventing components

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Configure PSA for the following Eventing components:

- NATS.
- Eventing controller.
- Eventing publisher.

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/15919

### CODE REVIEW

File: resources/eventing/values.yaml [hunk 1]
```diff
@@ -45,9 +45,37 @@ global:
...
+      - name: net.ipv4.ip_local_port_range
+        value: ''
+      - name: net.ipv4.ip_unprivileged_port_start
+        value: ''
+      - name: net.ipv4.tcp_syncookies
+        value: ''
+      - name: net.ipv4.ping_group_range
+        value: ''
+
+  containerSecurityContext:
```
Comment (mfaizanse) [inline resources/eventing/values.yaml:67]: Is NATS already running as non-root user? would now it would work with all NATS commands? storage etc.

Comment (marcobebway) [reply resources/eventing/values.yaml:67]: Yes, it is verified offline.

**Review by mfaizanse — APPROVED** [review]

**Pull request merged**